app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  belongs_to :user, class_name: 'User'
+  belongs_to :post
+
+  after_create :update_comments_counter
+
+  private
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your commit messages capitalize to make your pull request more professional 👍